tool/tctl/common/auth_command.go

```diff
@@ -505,11 +510,66 @@ func (a *AuthCommand) GenerateCRLForCA(ctx context.Context, clusterAPI authComma
 		return trace.Wrap(err)
 	}
 
-	crl, err := clusterAPI.GenerateCertAuthorityCRL(ctx, certType)
+	authorities, err := clusterAPI.GetCertAuthorities(ctx, certType, false)
 	if err != nil {
 		return trace.Wrap(err)
 	}
 
+	if a.output == "" {
+		l := len(authorities)
+		switch {
+		case l > 1:
+			return trace.BadParameter("found %d authorities, use --out to export all CRLs", l)
+		case l == 1 && len(authorities[0].GetActiveKeys().TLS) > 1:
+			return trace.BadParameter("CA has multiple active keys, use --out to export all CRLs")
+		}
```

Comment:
if there are multiple CAs returned from GetCertAuthorities it probably means there is a trusted cluster, may be a root or leaf, can we handle that and output the correct CRL? We shouldn't break this in trusted cluster setups where it's currently working

should probably also handle the case where it returns 0 authorities or there are 0 TLS keys